src/Entity/User.php

```diff
@@ -1208,6 +1208,13 @@ public function isInvitedToRelay(Centre $relay): bool
         return null !== $userRelay && !$userRelay->getBValid();
     }
 
+    public function hasValidLinkToRelay(Centre $relay): bool
+    {
+        $userRelay = $this->getSubjectRelaysForRelay($relay);
```

Comment:
Bad naming here, no ? Is is a UserRelay or a SubjectRelay, or a UserCentre ? We should discuss this